### PR TITLE
blind people know when things happen to them

### LINKED ISSUE #67815: Blind people do not get notifications for being stripped or hit by objects, or being forcefed (and probably a lot more)
## Reproduction:
Asking the clown to throw something at you, object goes pap, no notification.
Clown forces Greentext into your hands, and you get shoved for it, then someone takes your shoes. You get no notifications for it being put into your hands, or your shoes getting taken.
Security takes your wig while you're not paying attention and feeds it to you. You do not get a notification for being forcefed your wig.

### PR DESCRIPTION
**Pull request by Mooshimi**
## About The Pull Request

Fixes: #67815
Gives a vague chat alert to blind people who are being stripped of something/something is being put on them, thrown and hits them/hits them in general, being fed something

## Why It's Good For The Game

It's more realistic than not noticing at all if you're blind.

## Changelog

:cl:
fix: Blind support added to some actions like stripping, being hit, and force-feeding.
/:cl:

### CODE REVIEW

**Comment by Mooshimi** [comment]
will fix tomorrow don't yell at me

File: code/datums/elements/strippable.dm [hunk 1]
```diff
@@ -153,6 +157,7 @@
 	source.visible_message(
 		span_warning("[user] tries to remove [source]'s [item.name]."),
 		span_userdanger("[user] tries to remove your [item.name]."),
+		blind_message = "You hear rustling.",
```
Comment (Kapu1178) [inline code/datums/elements/strippable.dm:160]:
```suggestion
		blind_message = span_hear("You hear rustling."),
```

File: code/_onclick/item_attack.dm [hunk 2]
```diff
@@ -331,8 +331,10 @@
 		attack_message_spectator = "[user] [message_verb_continuous] [src][message_hit_area] with [I]!"
 		attack_message_victim = "[user] [message_verb_continuous] you[message_hit_area] with [I]!"
 	if(user == src)
-		attack_message_victim = "You [message_verb_simple] yourself[message_hit_area] with [I]"
+		attack_message_victim = "You [message_verb_simple] yourself[message_hit_area] with [I]."
 	visible_message(span_danger("[attack_message_spectator]"),\
 		span_userdanger("[attack_message_victim]"), null, COMBAT_MESSAGE_RANGE, user)
+	if(src.is_blind())
```
Comment (Mothblocks) [inline code/_onclick/item_attack.dm:337]:
```suggestion
	if(is_blind())
```

File: code/datums/elements/strippable.dm [hunk 3]
```diff
@@ -109,6 +110,9 @@
 					var/list/new_entry = list(list(user.name, "tried equipping you with [equipping]", world.time))
 					LAZYADD(victim_human.afk_thefts, new_entry)
 
+			else if(victim_human.is_blind()) //blind support
```
Comment (Mothblocks) [inline code/datums/elements/strippable.dm:113]: Don't need this comment, it's obvious from the `is_blind()`

```suggestion
			else if(victim_human.is_blind())
```

File: code/datums/elements/strippable.dm [hunk 4]
```diff
@@ -168,6 +173,9 @@
 				var/list/new_entry = list(list(user.name, "tried unequipping your [item.name]", world.time))
 				LAZYADD(victim_human.afk_thefts, new_entry)
 
+		else if(victim_human.is_blind())//blind support
```
Comment (Mothblocks) [inline code/datums/elements/strippable.dm:176]:
```suggestion
		else if(victim_human.is_blind())
```

File: code/modules/projectiles/projectile.dm [hunk 5]
```diff
@@ -306,13 +306,15 @@
 			playsound(loc, hitsound, 5, TRUE, -1)
 		else if(suppressed)
 			playsound(loc, hitsound, 5, TRUE, -1)
-			to_chat(L, span_userdanger("You're shot by \a [src][organ_hit_text]!"))
+			to_chat(L, span_userdanger("You're shot by \a [src][organ_hit_text]!"), )
```
Comment (Mothblocks) [inline code/modules/projectiles/projectile.dm:309]:
```suggestion
			to_chat(L, span_userdanger("You're shot by \a [src][organ_hit_text]!"))
```

**Review by Mothblocks — DISMISSED** [review]

**Comment by ZeroMODE0** [comment]
Bless

**Pull request merged**